### ISSUE TITLE
Improve discoverability of categories in the theme showcase

### ISSUE THREAD

**Issue by dsas** [Automattic/wp-calypso#96163]
### Quick summary

Some of the theme categories in the theme showcase are hidden behind a 'More' link. It would be easier to scan the page, and to discover all categories if they weren't hidden

https://github.com/user-attachments/assets/a8a46606-1d61-4c51-b2ec-604dfe986262

### Steps to reproduce

1. Go to wordpress.com/themes
2. Try to find the restaurant category

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

**Comment by Robertght** [comment]
I agree there's room for improvement here.

I don't know if we should keep this as a bug or enhancement, however, is @Automattic/t-rex able to handle this?

**Comment by dsas** [comment]
It needs design input before technical input, I don't think it's likely to be something we work on anytime soon though. There's a project going on to improve the filtering UI (p9Jlb4-fae-p2), but it has the same issue. cc @fditrapani

**Comment by fditrapani** [comment]
Thanks for the ping @dsas. I also agree that there's room for improvement here but I also don't see it happening soon. We have begun to diverge in experiences for what we offer in the design picker and the showcase. In the new year, we'll need to discuss how we tackle that. In addition to that, we also need to figure out how we'll untangle Calypso and wp-admin. So lots to happen here before we can address that. For that reason, I'd say we can close this as a won't fix and count on addressing it with future work.
